docs/RELEASE-NOTES.rst

```diff
@@ -15,6 +15,7 @@ Added Functionality
 * Support for virtual server source address translation configuration.
 * Added controller name and version to the metadata of certain BIG-IP LTM resources managed by the controller.
 * :issues:`433` - Support for pre-existing server ssl profiles for Ingresses.
+* Added the ability for users to customize the openshift route virtual servers by whitelisting them.
```

Comment:
Added support for attaching OpenShift Routes to existing BIG-IP virtual servers.

Reply:
Done